src/main/java/org/ow2/proactive/procci/model/utils/ConvertUtils.java

```diff
@@ -21,7 +21,7 @@
         try {
             return integerString.map(s -> Integer.parseInt(s));
```

Comment:
you could wrap this try and catch in a single function. Give me a shout if you need help